conda_build_all/tests/integration/test_inspect_binstar.py

```diff
@@ -74,6 +75,18 @@ def test_distribution_exists(self):
         # Check that the distribution has been added.
         self.assertTrue(distribution_exists_on_channel(CLIENT, OWNER, meta, channel='main'))
 
+        # Add the meta for a recipe known to exist on conda-forge
+        meta2 = self.write_meta('conda_build_all', """
+                                package:
+                                    name: conda-build-all
+                                    version: 0.12.0
+                                build:
+                                    script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
This line is superfluous and we could remove it if there are other review actions.